fecgov/openFEC
# 5365- Remove by_ofice and by_party endpoints

## Summary (required)

- Resolves #5365

Removes deprecated endpoints by_office and by_office/by_party

### Required reviewers

1-2 dev

## Impacted areas of the application

General components of the application that this PR will affect:

-  API

## Screenshots




## How to test

- Activate your local pyenv
- pull this branch
- pytest
- See that candidates/totals/by_office and by_office/by_party have been removed. Run a few queries using the new endpoint candidate/totals/aggregates and use aggregate_by office-party
